File: InAppPurchaseV3SpikeProject/InAppPurchaseV3Spike/src/main/java/com/novoda/inapp/v3/MainActivity.java
package com.novoda.inapp.v3;

import android.os.Bundle;
import android.support.v7.app.ActionBarActivity;

public class MainActivity extends ActionBarActivity {

    /**
     * I'm up to here : http://developer.android.com/training/in-app-billing/purchase-iab-products.html
     */

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .add(R.id.container, new MainFragment())
                    .commit();
        }
    }

}

File: dungeon-crawler/companion/src/main/java/com/novoda/dungeoncrawlercompanion/MainActivity.kt
package com.novoda.dungeoncrawlercompanion

import android.os.Bundle
import android.support.design.widget.BottomNavigationView
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        navigation.setOnNavigationItemSelectedListener(mOnNavigationItemSelectedListener)
        navigation.selectedItemId = R.id.navigation_game
    }

    private val mOnNavigationItemSelectedListener = BottomNavigationView.OnNavigationItemSelectedListener { item ->
        when (item.itemId) {
            R.id.navigation_game -> {
                supportFragmentManager.beginTransaction().replace(R.id.content, GameFragment()).commit()
                return@OnNavigationItemSelectedListener true
            }
            R.id.navigation_about -> {
                supportFragmentManager.beginTransaction().replace(R.id.content, AboutFragment()).commit()
                return@OnNavigationItemSelectedListener true
            }
        }
        false
    }
}
